api/structure/models.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import Column, String, Integer
+
+from utilities.utility import Utility
+from helpers.database import Base
+
+
+class Structure(Base, Utility):
+    __tablename__ = "office_structures"
+
+    id = Column(Integer, primary_key=True)
+    # web_id is the id from the client side
+    web_id = Column(String, nullable=False)
+    level = Column(Integer, nullable=False)
+    name = Column(String(50), nullable=False)
+    parent_id = Column(Integer, nullable=False)
+    tag = Column(String(50), nullable=False)
```

Comment:
About this tag column. Kindly confirm that it is a `nullable=False`.

Reply:
The front-end dev indicated that all the fields are required @nakatuddesuzan

Reply:
Noted